### PR TITLE
Layer diagram improvements

### PR DESCRIPTION
**Pull request by portsmouth**
Improved this diagram a bit:

- expanded the glossy-diffuse slab to represent it as layer of clear-coat dielectric gloss on diffuse, as it is defined in the text. 
- indicated the transparent slabs that have embedded volume (i.e. all except the gloss).
- the number of slabs (i.e. the boxes) is now 8, matching the formal structure of the model (see bottom)
- also there are 5 transparent slabs, corresponding to the 5 slabs with a VDF in the formal structure.

image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -855,17 +853,26 @@
...
+In the full light transport, the observed color of the coated base is darkened and saturated due to multiple internal reflections from the inside of the coat, which causes light to strike the underlying material multiple times and undergo more absorption, and the observed **`coat_color`** tint also darkens as the incidence angle changes due to the change in path length in the medium. Also, the presence of a rough coat will increase the apparent roughness of the BSDF lobes of the underlying base. We assume that in the ground truth appearance, all these physical effects are accounted for. In the following sub-sections, we detail recommendations for implementation of them.
+
+### Roughening
```
Comment (virtualzavie) [inline index.html:863]: Did you intend to include the coat roughening sub-section in this PR?

File: images/layer_graph.png
[the remark this answers is not in the document]
Comment (virtualzavie) [reply images/layer_graph.png:1]: Re-reading the spec, I see the diffuse slab is specified differently from the translucent and subsurface one, where the diffuse slab only has a diffuse BRDF, whereas the other two have a dielectric specular BRDF and a volume.

File: index.html (hunk 1 above)
Comment (portsmouth) [reply index.html:863]: It's done in a separate PR, which is ready for merge:  

https://github.com/AcademySoftwareFoundation/OpenPBR/pull/172

**Review by virtualzavie — APPROVED** [review]
LGTM

**Comment by portsmouth** [comment]
Merged main, ready to merge.

**Review by jstone-lucasfilm — APPROVED** [review]
This looks good to me, thanks @portsmouth.

**Pull request merged**